src/openms/source/ANALYSIS/RNPXL/RNPxlModificationsGenerator.cpp

```diff
@@ -441,17 +441,17 @@ RNPxlModificationMassesResult RNPxlModificationsGenerator::initModificationMasse
 }
 
 //static
-void  RNPxlModificationsGenerator::generateTargetSequences(const String& res_seq, Size pos, const map<char, vector<char> >& map_source2target, StringList& target_sequences)
+void  RNPxlModificationsGenerator::generateTargetSequences(const String& res_seq, Size param_pos, const map<char, vector<char> >& map_source2target, StringList& target_sequences)
```

Comment:
Yes sorry, tests will follow asap. 
These functions are only used in a util, placed in subfolder and under quite heavy developement.
See e.g.
https://github.com/timosachsenberg/OpenMS/tree/feature/RNPxlSearch_postscore
Hope that somehow justifies my lazyness to create the tests earlier.